### ISSUE TITLE
Calculating exposure to PM2.5

### ISSUE THREAD

**Issue by markotainio** [ITHIM/ITHIM-R#19]
Discussion on how the exposure to air pollution is calculated. This relates to following two Issues:

Exposure concentration of PM2.5 while in traffic
https://github.com/ITHIM/ITHIM-R/issues/13

Ventilation rate and physical activity
https://github.com/ITHIM/ITHIM-R/issues/3

--
Overall, the idea here is to estimate inhaled dose of PM2.5 by taking into account:
- Background concentration of the study area
- Concentration of PM2.5 while in traffic (Issue 13)
- Ventilation rate while doing physical activity (any physical activity) (Issue 3)

Attached Excel will illustrate the calculation, including how to exposure is later converted to RRs.

[ITHIM-R Air pollution exposure.xlsx](https://github.com/ITHIM/ITHIM-R/files/1724518/ITHIM-R.Air.pollution.exposure.xlsx)

**Comment by leandromtg** [comment]
@markotainio, I checked the Excel file and I am not sure whether the way you split the activities will work. I think we are not going to be able to know the type of PA or where they occurred in 90+% of the cases. It is not a common question in surveys, unless it is a PA-specific survey. Most probably we are going to have only broad categories of domain (leisure, transportation and work, sometimes household) and/or intensity. In these cases we will probably use an average MET expected for each category, but no clue of where activities might have been happened (specially the leisure and work ones).

Work activities are under the “Rest” category, right? Not sure if it is really relevant here, but most of the light-intensity work activities indeed occur indoors (although there are exceptions, like truck and public transport drivers); however, moderate-to-vigorous work activities, when ventilation rate is higher, might occur in more varied environments.

**Comment by markotainio** [comment]
@leandromtg We don't need to know where these activities happen, only how they impact ventilation rate. As you can see from Excel, I did not assume any location specific changes in PM2.5 concentration in Gym, and I would not model any location specific changes in other PA locations either. We can do same calculation e.g. for sport or leisure by assuming background PM2.5 concentration and ventilation rate based on MET.

What I do not want to do is to take ventilation rate into account only for transport environment. It will create situation where we treat ventilation rate differently for different PA categories, and that it not a good way to do it. 

Work is ignored in these calculations consistently. So far all PA we take into account is non-occupational PA so therefore it is ignored also here. If this changes, then we can add occupational PA and related exposure also here.

**Comment by leandromtg** [comment]
So "Gym" does not refer to type of activity or place, but to leisure-time physical activity in general. Perhaps we should then change the term to avoid future confusions.

Agree with you about being consistent across PA domains whenever is possible.

**Comment by markotainio** [comment]
Yes, "gym" is bad name for the example. Gives impression that we know more than we do.

**Comment by markotainio** [comment]
New review article relating to the topic: Air pollution as a risk factor in health impact assessments of a travel mode shift towards cycling
http://www.tandfonline.com/doi/full/10.1080/16549716.2018.1429081

**Comment by gotom22** [comment]
@markotainio @leandromtg 
[HEAT](www.heatwalkingcycling.org) approach could be of interest. It basically makes one simple spatial distinction "in traffic" vs. "away from traffic". 
(In HEAT, counterfactual - i.e. trip not taken - is sometimes unspecified. Then air pollution level is assigned based on trip **purpose**: active travel for transport is compared to "in traffic", whereas active travel trips for leisure are compared to background concentrations. )
In ITHIM reference and counterfactual (or "both scenarios") are always specified, I would think. )

**Comment by gotom22** [comment]
I suggest we define some terminology/structure around "spatial aggregation" (for air pollution, but maybe more generally). It was my impression ideas vary widely.... (@neil not on GH yet?). My sense 4 below is out of scope, not sure about 3. 

**Spatial aggregation levels**
1. City (background)
2. In traffic/away from traffic (background*ratio)
3. Home location buffer 
-- based on proximity to major road 
-- based on dispersion model
4. Cumulative exposure
-- incl. on route (based on travel demand model and dispersion model)

**Comment by markotainio** [comment]
HEAT approach is pretty much same approach as here. If we know that cycling and walking is for transport, then higher concentration. Otherwise background concentration.

Not sure what you mean with spatial aggregation part? I think in most cases we have on estimate for city background and then travel mode specific concentrations. In some special cases we have more information.

**Comment by gotom22** [comment]
@markotainio 
re spatial aggregation levels:
my recollection may be off - but didn't Neil have ideas around route specific exposure estimates - i.e. overlaying travel model outputs with AP dispersion models...? (I would think this is out of scope). 

However, it might be useful to define some common terminology...so discussions stay focused.
